packages/components/src/styles/components/form/super-select.scss

```diff
@@ -0,0 +1,490 @@
+    box-shadow: var(--token-elevation-low-box-shadow);
+
+    &:hover,
+    &.mock-hover {
+      border-color: var(--token-form-control-base-border-color-hover);
+    }
+
+    &:focus,
+    &.mock-focus {
+      border-color: var(--token-color-focus-action-internal);
+      outline: 3px solid var(--token-color-focus-action-external);
+      outline-offset: 0;
+    }
+
+    &[aria-expanded="true"] {
+      &.ember-basic-dropdown-trigger--in-place {
+        border-radius: var(--token-form-control-border-radius);
+      }
+    }
+
+    // Hide default trigger dropdown arrow
+    .ember-power-select-status-icon {
+      display: none;
+    }
+
+    // Trigger content (Single Select):
+    .ember-power-select-placeholder,
+    .ember-power-select-selected-item {
+      flex: 1;
+      margin-left: 0;
+    }
+
+    .ember-power-select-placeholder {
+      color: var(--token-form-control-base-foreground-placeholder-color);
+    }
+
+    &[aria-disabled="true"] {
+      color: var(--token-form-control-disabled-foreground-color);
+      background-color: var(--token-form-control-disabled-surface-color);
+      background-image: var(--token-form-select-background-image-data-url-disabled);
+      border-color: var(--token-form-control-disabled-border-color);
+      box-shadow: none;
+      cursor: not-allowed;
+      opacity: 1;
+
+      .ember-power-select-selected-item {
+        opacity: inherit;
+      }
+    }
+  }
+
+  // DROPDOWN
+  .ember-basic-dropdown-content {
+    max-width: var(--hds-form-super-select-dropdown-max-width);
+    margin: 0;
+    border: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+    border-radius: var(--token-form-control-border-radius);
+
+    &.ember-basic-dropdown-content--in-place {
+      box-shadow: var(--token-elevation-high-box-shadow), var(--token-elevation-high-box-shadow);
+    }
+
+    &.ember-basic-dropdown-content--below {
+      margin-top: 4px;
+    }
+
+    &.ember-basic-dropdown-content--above {
+      top: unset !important; // override inline style (TODO: update in JS)
+      bottom: 100%;
+      margin-bottom: 4px;
+    }
+
+    // hide first group separator line
+    > .ember-power-select-options > .hds-form-super-select__option-group:first-child::before {
+      display: none;
+    }
+  }
+
+  // Options List
+  .ember-power-select-options {
+    padding: 3px 0;
+
+    .ember-power-select-options {
+      padding: 0;
+    }
+  }
+
+  // Options Item
+  .ember-power-select-option {
+    position: relative;
+    min-height: $hds-super-select-item-height;
+    padding: 7px 7px 7px 39px; // top, right, bottom, left
+    line-height: var(--token-typography-body-200-line-height);
+    overflow-wrap: break-word; // when used with flex parent, content may overflow (therefore not using flex layout on parent)
+    border: none; // guard against border in default theme
+
+    + .ember-power-select-option {
+      margin-top: 8px;
+    }
+
+    // states
```

Comment:
I believe @KristinLBradley was looking into this – but correct me if I'm wrong

Reply:
@alex-ju Sorry for some reason I thought that I had seen an example you added. I modified the examples now to make one of the options disabled.

(I can add more examples if it would be helpful.)